src/java/org/apache/cassandra/fql/FullQueryLogger.java

```diff
@@ -117,6 +120,30 @@ public synchronized void enable(Path path, String rollCycle, boolean blocking, i
         }
     }
 
+    public FullQueryLoggerOptions getFullQueryLoggerOptions()
+    {
+        if (isEnabled())
+        {
+            final FullQueryLoggerOptions options = new FullQueryLoggerOptions();
+            final BinLogOptions binLogOptions = binLog.getBinLogOptions();
+
+            options.archive_command = binLogOptions.archive_command;
```

Comment:
@dcapwell tell me if you want a static method in FQLO to do this, I am fine with this one but everybody has different taste when it comes to coding, I cant guess how you code.

Reply:
since this is the only place that converts between the two, I feel that this is fine.